README.md

```diff
@@ -170,20 +170,19 @@ Flags:
 
 ## Roadmap
 
-* Additional language support for just-in-time (JIT) compilers, and dynamic languages (non-exhaustive list):
-  * Ruby
-  * Node.js
-  * Python
+* Additional language support for just-in-time (JIT) compilers and dynamic languages (non-exhaustive list):
   * JVM
+  * Lua
+  * Node.js
```

Comment:
We technically support it too, no?

Reply:
I would say we err on the side of "if it's possible at all we say we support it". So in that sense we do support JVM and nodejs. Not completely 100% zero-instrumentation yet, but with a tiny bit of effort people can already get it to work.